manifests/server.pp

```diff
@@ -297,6 +298,23 @@
     }
   }
 
+  if $automatic_dlo_cleanup {
+    if $facts['systemd'] {
+      # deploy a systemd timer + service to cleanup old reports
+      # https://puppet.com/docs/puppetdb/5.2/maintain_and_tune.html#clean-up-the-dead-letter-office
+      systemd::unit_file{'puppetdb-cleanup.service':
+        content => epp("${module_name}/puppetdb-DLO-cleanup.service.epp"),
+      }
+      -> systemd::unit_file{'puppetdb-cleanup.timer':
+        content => epp("${module_name}/puppetdb-DLO-cleanup.timer.epp"),
+      }
+      -> service{'puppetdb-cleanup.timer':
+        ensure => 'running',
+        enable => true,
```

Comment:
I don't think this is necessary, but I could be mistaken.

Reply:
It needs to be enabled. Otherwise the timer is off after the next reboot and the upcoming puppet run would start it.